memcpy_audio.S

```diff
@@ -85,7 +85,18 @@
 	pkhtb r6,r8,r7,ASR #16
 
 	stmia r0!, {r3,r4,r5,r6}
-	pop	{r4-r8,r14}	
+	pop	{r4-r8,r14}
```

Comment:
I actually don't believe need to push & pop `r14` for `AUDIO_BLOCK_SAMPLES == 8` or `AUDIO_BLOCK_SAMPLES == 4`.